### PR TITLE
feat: add additional include sections and email param

### PR DESCRIPTION
**Pull request by P1sar**

### CODE REVIEW

File: includes/hiring_steps_default.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: plain
+---
+
+
+### Hiring Steps
```
Comment (morrigan) [inline includes/hiring_steps_default.md:5]: Is this something that was left over accidentally? Doesn't seem to have much sense and it's unused so better to remove

File: includes/hiring_steps_gossamer_extended.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+### Hiring Steps
+
+- Selected candidates will be invited to a 30–to–45–minute screening call with one of our tech recruiters
+- Next, candidates will be invited to a 60-minute values interview with one of our team members
+    - Given they pass the values interview, we ask them to complete the first technical assignment.  Based on the submission we can determine if we want to continue with the interview process.
+    - **NOTE**: If candidates have verifiable  OpenSource work experience this step might be skipped
+- Given the review of the first technical assignment, a technical 60-minute interview with our engineers will be scheduled.
+    - Among other follow up question based on the first technical assignment will be asked during technical interview to prove they are the author of submission and to see how they can code/problem solve in a live environment.
+- Then, if necessary, candidates will be asked to complete a second technical task in under 10-20 hours. Your time and expertise demonstrated in the work test challenge will be compensated upon completion.
+- Your time and expertise demonstrated in the work test challenge will be compensated
```
Comment (morrigan) [inline includes/hiring_steps_gossamer_extended.md]: redundant with previous point (same sentence)

File: includes/hiring_steps_default.md (hunk 1 above)
Comment (P1sar) [reply includes/hiring_steps_default.md:5]: Nope, I want to have possibility to setup different hiring steps for different positions/projects. Bcs as you can see in `includes/hiring_steps_gossamer_extended.md` we have extended hiring steps for current role

File: includes/contact.md [hunk 3]
```diff
@@ -7,13 +7,6 @@ layout: plain
 If you feel like you're the perfect fit for the role, please apply even if you
 don't match all the requirements.
 
-### Hiring Steps
```
Comment (P1sar) [inline includes/contact.md:10]: @morrigan I have basically removed hiring steps from contact and made hiring steps also to be includable

**Review by morrigan — APPROVED** [review]

**Pull request merged**